File: RssReaderObj/RssReaderObj/RssItem.m
//
//  RssItem.m
//  RssReaderObj
//
//  Created by Admin on 11.10.16.
//  Copyright © 2016 Admin. All rights reserved.
//

#import "RssItem.h"

@implementation RssItem
-(instancetype)init
{
    _title = [NSMutableString stringWithString:@""];
    _imageUrl = [NSMutableString stringWithString:@""];
    _link = [NSMutableString stringWithString:@""];

    _pubDate = [NSMutableString stringWithString:@""];

    return self;
}
-(void)print
{
    NSLog(@"%@ %@ %@ %@",_title,_imageUrl,_link,_pubDate);
}
-(RssItem*)createcopy
{
    RssItem *newItem=[[RssItem alloc] init];
    newItem.title = [self.title copy];
    newItem.link = [self.link copy];
    newItem.pubDate = [self.pubDate copy];
    newItem.imageUrl = [self.imageUrl copy];
    newItem.imageData = [self.imageData copy];
    return newItem;
}
@end

File: MechanicReader/MechanicReader/RssItem.swift
//
//  RssItem.swift
//  MechanicReader
//
//  Created by Admin on 10.10.16.
//  Copyright © 2016 Admin. All rights reserved.
//

import Foundation

public class RssItem
{
    var title:String
    var link: String
    var pubDate:String
    var imageUrl:String
    var image:NSData?
    
  public  init()
    {
        self.title = ""
        self.link = ""
        self.pubDate = ""
        self.imageUrl=""
    }
}